### PR TITLE
Create UI

### PR DESCRIPTION
**Pull request by vflorin1995**
### Changes:

-Create UI for calculator

### CODE REVIEW

File: src/Components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+          <button type="button"> 5 </button>
+          <button type="button"> 6 </button>
+          <button type="button" className="orange"> - </button>
+          <button type="button"> 1 </button>
+          <button type="button"> 2 </button>
+          <button type="button"> 3 </button>
+          <button type="button" className="orange"> + </button>
+          <button type="button" className="zero"> 0 </button>
+          <button type="button"> . </button>
+          <button type="button" className="orange"> = </button>
```
Comment (FrederickMih) [inline src/Components/Calculator.js:28]: - [**Optional**] I think these lines of code are repeating and bulky. We can make things look clean and clear by declaring an array of all these numbers and characters, iterating over them, and displaying a single button.
You can get an idea of what I'm saying from the piece of code below:

```.js
   calButtons = [
      'AC', '+/-', '%', '/',
      '7', '8', '9', '*',
      '6', '5', '4', '-',
      '3', '2', '1', '+',
      '0', '.', '=',
    ];
    
    ...

    calButtons.map((btn) => (
            <button >
              {btn}
            </button>
```

File: src/index.js [hunk 2]
```diff
@@ -1,17 +1,11 @@
...
+import Calculator from './Components/Calculator';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
-    <App />
+    <Calculator />
   </React.StrictMode>,
 );
-
```
Comment (FrederickMih) [inline src/index.js]: - [ ] Seems you are missing the App component in your src directory. Please you need this component. Based on the requirements of this project, the App component should have the imported Calculator.js component, not the index.js component and the App component should be class-based. I think you'll require to fix :wrench: this issue and move a step toward the approval of this milestone.

File: src/Components/Calculator.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import './Calculator.css';
+
+class Calculator extends React.PureComponent {
+  render() {
+    return (
+      <div>
+        <div className="display" />
```
Comment (FrederickMih) [inline src/Components/Calculator.js]: - [ ] On the design given to you, the display shows Zero(0). Your design is not the same. You'll require to fix :wrench: it.

| Original | Yours |
| --- | --- |
| Screenshot from 2022-10-26 11-01-45 | Screenshot from 2022-10-26 11-02-24 |

**Review by FrederickMih — CHANGES_REQUESTED** [review]
Hi @vflorin1995 , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!


Screenshot from 2022-10-20 09-38-51


## Highlights :green_circle: 

- Descriptive README.md file :+1: 
- Correct Gitflow :heavy_check_mark: 
- No Linter error :+1: 

## Required Changes ♻️

- [ ]  Based on the requirements of this project, all the boilerplates that got created by the CRA(texts, images, and styles) ought to have gotten deleted. The public folder still contains these boilerplates. Let's get rid of them. This will render your codebase clean.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AnselemOdims — APPROVED** [review]
### Hi  @vflorin1995 🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**